js/damage.js

```diff
@@ -305,6 +305,8 @@ function getDamageResult(attacker, defender, move, field) {
         bpMods.push(0x800);
         description.moveBP = move.bp / 2;
         description.weather = field.weather;
+    } else if (defender.weight <= "200" && move.name === "Sky Drop" || move.name === "Sky Drop" && (defender.type1 === "Flying" || defender.type2 === "Flying")) {
```

Comment:
Several issues here, not least of which is this doesn't work at all.
- This should be done in `getDamageResult()` instead
- 200 shouldn't be a string
- You should be checking if the weight is >=, not less than

Basically `(move.name === "Sky Drop" && (defender.weight >= 200 || (defender.type1 === "Flying" || defender.type2 === "Flying")))` would be a better check.